### DISCUSSION TITLE
[Q&A]

### DISCUSSION THREAD

**Discussion by Gozilasim** [NVIDIA/NVFlare#3066] [Q&A]
### Python version (`python3 -V`)

3.8.10

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.5.0

### NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)

2.5

### Operating system

Ubuntu 20.04.6LTS

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [ ] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

from nvflare.fuel.utils.gpu_utils import get_host_gpu_ids, get_host_gpu_memory_total

finally i found out the problems, this library cannot detect gpu inside the jetson orin , that's why i cannot allocate gpu and memory on each site, how to solve , please help ,tqvm

**Comment by YuanTingHsieh** [comment]
@Gozilasim closing this one, please refer to the other discussion thread thanks!
